input/fsh/profiles/FinnishSchedulingLocation.fsh

```diff
@@ -0,0 +1,39 @@
+Profile: FinnishSchedulingLocation
```

Comment:
There's probably nothing scheduling specific here?
We should change to use the profile from base profiles.